express-rest-crud-mvc-index/lesson-notes/README.md

```diff
@@ -13,7 +13,8 @@ We are going to be building this app over several sessions, so let's make sure w
 - `touch .gitignore` (tell git which files to ignore)
 - `touch app.js`
 - `touch .env`
-- `npm install express dotenv`
+- `npm install express dotenv nodemon`
```

Comment:
Did your class opt not to install `nodemon` globally?

If you did not install it globally, is it worth/do you think the students could handle `npm install nodemon --save-dev` or is it better to keep it simple as one line?